openproblems/tasks/dimensional_reduction/datasets/10x_pbmc.py

```diff
@@ -0,0 +1,8 @@
+  
+from ....data._10x_5k_pbmc import load_10x_5k_pbmc
```

Comment:
This shouldn't be a private submodule -- can we remove the underscore?

Reply:
Looks like the file is now totally deleted -- did you forget a git add? also the reference to it here should not have an underscore.

Reply:
Fixed - got called into a meeting and didn't finish what I was doing lol